sentry/src/main/java/io/sentry/SentryLogEvent.java

```diff
@@ -0,0 +1,182 @@
+package io.sentry;
+
+import static io.sentry.DateUtils.doubleToBigDecimal;
+
+import io.sentry.protocol.SentryId;
+import io.sentry.vendor.gson.stream.JsonToken;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class SentryLogEvent implements JsonUnknown, JsonSerializable {
+
+  private @NotNull SentryId traceId;
+  private @NotNull Double timestamp;
+
+  private @NotNull String body;
+
+  private @Nullable SentryLevel level;
+  private @Nullable Map<String, SentryLogEventAttributeValue> attributes;
+  private @Nullable Map<String, Object> unknown;
```

Comment:
I think the optional `severity_number` is missing which later on is inferred by the SDK, e.g 

https://github.com/getsentry/sentry-javascript/blob/caa13167c5d6e35dd2a5dbc08b86f0f0f40681eb/packages/core/src/logs/constants.ts#L8

Reply:
Do you mean for other SDKs using the Java SDK? That should land in `unknown` if Java SDK doesn't have it directly.

I can add it in a follow up PR.

Reply:
There is also the severity number field that is optional. If it is not set (by the user) then  the SDK will use the default values like in the link above

See [here](https://github.com/getsentry/sentry-javascript/blob/develop/packages/core/src/types-hoist/log.ts#L30)